### PR TITLE
Add Invalid Org Discovery Type Error Message

### PR DESCRIPTION
**Pull request by dhaura**
### Purpose
- Add error message support for invalid organization discovery type in email domain based organization discovery.

### Related Issues
- https://github.com/wso2/product-is/issues/20953

### Related PRs
- https://github.com/wso2-extensions/identity-auth-organization-login/pull/45

### Checklist

- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties [hunk 1]
```diff
@@ -243,8 +243,8 @@ authentication.context.null=Pardon! Nous avons dû interrompre votre connexion
 authentication.context.null.description=Cela se produit généralement en raison dun long retard ou dune action irrégulière dans votre processus de connexion. Recommençons et réessayons de nous connecter. Si le problème persiste, veuillez contacter votre administrateur.
 authentication.flow.timeout=Pardon! Votre tentative de connexion a pris trop de temps
 authentication.flow.timeout.description=Il semble que le processus de connexion s'est terminé en raison du retard. Recommençons. Si ce problème persiste, veuillez contacter l'administrateur.
-authentication.flow.app.disabled=D�sol�! L'application est d�sactiv�e.
-authentication.flow.app.disabled.description=Il semble que l'application soit d�sactiv�e et que l'acc�s � l'application soit restreint. Veuillez contacter votre administrateur.
+authentication.flow.app.disabled=Désolé! L'application est désactivée.
```
Comment (DilshanSenarath) [inline identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties:246]: Is this change expected?

File: identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties [hunk 2]
```diff
@@ -502,6 +502,7 @@ organization.login=Connexion de l'organisation
 organization.name=Nom de l'organisation
 organization.email=E-mail de l'organisation
 invalid.organization.discovery.input=Impossible d'identifier l'organisation. Veuillez fournir le nom de l'organisation pour continuer.
+invalid.organization.discovery.type=Le type de découverte d’organisation donné n’est pas valide ou n’est pas activé.
```
Comment (DilshanSenarath) [inline identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties:505]: I noticed some unusual characters in this text. Could you double-check to ensure everything is correct? If you're using IntelliJ to edit the text, you might want to try using VS Code instead. Also, please check other languages as well.

Comment (pavinduLakshan) [reply identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_fr_FR.properties:505]: Unusual characters in the github diff should be due to Github using UTF-8 for the UI and some french characters need different encoding. That should be expected.

**Comment by dhaura** [comment]
- The objective of this PR is achieved with https://github.com/wso2/identity-apps/pull/6803.

**Pull request closed without merging**